app/utils/bitschema_utils.py

```diff
@@ -597,3 +597,167 @@ class Timezone(Enum):
+    HISPANIC = "Hispanic/Latinx"
+    NATIVE_AMERICAN = "Native American/Alaska Native/First Nations"
+    MIDDLE_EASTERN = "Middle Eastern/North African (MENA)"
+    ASIAN = "Asian"
+    OTHER = "Other"
+    DECLINED = "Prefer not to say"
+    NOT_APPLICABLE = "Not Applicable"
+
+    def ethnicity(self):
+        return list(map(str, self))
+
+
+@unique
+class SexualOrientation(Enum):
+    HETEROSEXUAL = "Heterosexual/Straight"
+    LGBTIA = "LGBTIA+"
+    OTHER = "Other"
+    DECLINED = "Prefer not to say"
+    NOT_APPLICABLE = "Not Applicable"
+
+    def sexualOrientation(self):
+        return list(map(str, self))
+
+
+@unique
+class Religion(Enum):
+    CHRISTIANITY = "Christianity"
+    JUDAISM = "Judaism"
+    ISLAM = "Islam"
+    HINDUISM = "Hinduism"
+    BUDDHISM = "Buddhism"
+    OTHER = "Other"
+    DECLINED = "Prefer not to say"
+    NOT_APPLICABLE = "Not Applicable"
+
+    def religion(self):
+        return list(map(str, self))
+
+
+@unique
+class PhysicalAbility(Enum):
+    WITH_DISABILITY = "With/had limited physical ability (or with/had some type of physical disability/ies)"
+    WITHOUT_DISABILITY = "Without/have no limitation to physical ability/ies"
+    OTHER = "Other"
+    DECLINED = "Prefer not to say"
+    NOT_APPLICABLE = "Not Applicable"
+
+    def physicalAbility(self):
+        return list(map(str, self))
+
+
+@unique
+class MentalAbility(Enum):
+    WITH_DISORDER = "With/previously had some type of mental disorders"
+    WITHOUT_DISORDER = "Without/have no mental disorders"
+    OTHER = "Other"
+    DECLINED = "Prefer not to say"
+    NOT_APPLICABLE = "Not Applicable"
+
+    def mentalAbility(self):
+        return list(map(str, self))
+
+
+@unique
+class SocioEconomic(Enum):
+    UPPER = "Upper class/Elite"
+    UPPER_MIDDLE = "Upper Middle class (or High-level Professionals/white collars e.g. enginers/accountants/lawyers/architects/managers/directors"
+    LOWER_MIDDLE = "Lower Middle class (e.g. blue collars in skilled trades/Paralegals/Bank tellers/Sales/Clerical-Admin/other support workers)"
+    WORKING = "Working class (e.g. craft workers factory labourers restaurant/delivery services workers"
+    BELOW_POVERTY = "Underclass working but with wages under poverty line receiving Social Benefit from Government"
+    OTHER = "Other"
+    DECLINED = "Prefer not to say"
+    NOT_APPLICABLE = "Not Applicable"
+
+    def socioEconomic(self):
+        return list(map(str, self))
+
+
+@unique
+class HighestEducation(Enum):
+    BELOW_HIGH_SCHOOL = "Have/did not completed High School"
+    HIGH_SCHOOL = "High School Diploma"
+    ASSOCIATE = "Associate Degree"
+    BACHELOR = "Bachelor's Degree"
+    MASTER = "Master's Degree"
+    PHD = "PhD or other Doctorate Degrees"
+    OTHER = "Other"
+    DECLINED = "Prefer not to say"
+    NOT_APPLICABLE = "Not Applicable"
+
+    def highestEducation(self):
+        return list(map(str, self))
+
+
+@unique
+class YearsOfExperience(Enum):
+    UNDER_ONE = "Less than a year"
```

Comment:
@epicadk , the reason I did it like this (in word instead of number) is because IMO it makes more sense when an organization open a mentoring program opportunity, they can state the target candidate based on a range of years of experience (e.g. between 3 to 5 years) instead of 1,2,3,4 yr etc. This is also to make it easier for later when we make the logic which allows user to filter mentoring program based on certain categories (e.g. years of experience required "between 3 - 5 years" instead of selecting a particular number like 1,2,3 or 4 yr etc. Hope this makes sense 😉 .
![image](https://user-images.githubusercontent.com/29667122/117103206-3edc9600-adbd-11eb-97eb-24b04d891e18.png)

Reply:
We can still make the same query with databases ( possibly more optimized as well if we use indexing) further every year we could just ++ the count in the database rather than having to migrate the strings.

Reply:
@epicadk , what do you mean by  
> further every year we could just ++ the count in the database rather than having to migrate the strings.

Do you mean we need to make a scheduler to automatically add one year to the years of experience of each users who have this data instead of users manually updating their years of experience from their end? We could do that I suppose, but we're at disadvantage when the user is no longer in the industry but hasn't cancelled their membership/delete their account with us as that would be an inefficient use of overhead, plus we don't know how many of our users are in this category 🤔 .

Let's wait for @isabelcosta's opinion on this 😉 .